<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDlg</class>
 <widget class="QDialog" name="settingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>You</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="player1Name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maxLength">
           <number>32</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Match settings</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Match length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="matchLength">
          <property name="toolTip">
           <string>Choose the length of the match in points</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;In Greece, tables games are called &lt;span style=&quot; font-weight:600;&quot;&gt;tavli&lt;/span&gt; (related to the word &lt;span style=&quot; font-style:italic;&quot;&gt;tavla&lt;/span&gt;, meaning &quot;board&quot; or &quot;table&quot;, and &lt;a href=&quot;http://en.wikipedia.org/wiki/Cognate&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;cognate&lt;/span&gt;&lt;/a&gt; to the Latin &lt;span style=&quot; font-style:italic;&quot;&gt;tabula&lt;/span&gt;). There are three major variants, &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;portes&lt;/span&gt;, &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;plakoto&lt;/span&gt;, and &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;fevga&lt;/span&gt;. Portes resembles &lt;a href=&quot;http://en.wikipedia.org/wiki/Backgammon&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;backgammon&lt;/span&gt;&lt;/a&gt;, with minor variations: there is no doubling cube, and a backgammon counts only as a gammon (called &lt;span style=&quot; font-style:italic;&quot;&gt;diplo&lt;/span&gt;, Greek for &lt;span style=&quot; font-style:italic;&quot;&gt;double&lt;/span&gt;). Plakoto is very similar to mahbusa or &lt;a href=&quot;http://en.wikipedia.org/wiki/Tapa_%28game%29&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;tapa&lt;/span&gt;&lt;/a&gt;, while fevga is similar to narde or the Turkish variant &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;moultezim&lt;/span&gt;. The three are normally played consecutively, in three-, five- or seven-point matches.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; font-weight:600;&quot;&gt;Source: Wikipedia&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="specialValueText">
           <string>1 point</string>
          </property>
          <property name="suffix">
           <string> points</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>7</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_portes">
          <property name="text">
           <string>Portes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_plakoto">
          <property name="text">
           <string>Plakoto</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_fevga">
          <property name="text">
           <string>Fevga</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Opponent</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Network</string>
          </property>
          <property name="icon">
           <iconset resource="tavli.qrc">
            <normaloff>:/images/network.png</normaloff>:/images/network.png</iconset>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remoteIP"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Computer</string>
        </property>
        <property name="icon">
         <iconset resource="tavli.qrc">
          <normaloff>:/images/server.png</normaloff>:/images/server.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>211</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tavli.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>remoteIP</sender>
   <signal>textChanged(QString)</signal>
   <receiver>settingsDlg</receiver>
   <slot>network_textchanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>network_textchanged(QString)</slot>
 </slots>
</ui>
